src/main/java/ar/edu/unq/desapp/grupoO022020/backenddesappapi/model/Project.java

```diff
@@ -3,68 +3,87 @@
 import java.util.Date;
 
 public class Project {
-	
-	private Integer factor; 
+	private Integer factor;
 	private Float percentage;
 	private String name;
 	private Date endDate;
 	private Date startDate;
 	private Location location;
 	private Float amountCollected;
-	
-	public Project(String name, Date endDate, Date startDate, Location location) {
+
+	public Project(String name, Date endDate, Date startDate, Location location, Float amount) {
 		this.factor = 1000; // de 0 a $100.000
 		this.percentage = 1F; // Porcentaje mínimo de cierre de proyecto: de 50% a 100%
 		this.name = name;
 		this.endDate = endDate;
 		this.startDate = startDate;
 		this.location = location;
-		this.amountCollected = 0F;
+		this.amountCollected = amount;// esto solo es para el caso de poder validar los test pero el valor deberia ser
```

Comment:
@MaarLop , y no debería quedar con 0F devuelta? ... Se podría usar el ProjectBuilder sino?

Reply:
Pasa que en builder haces un new, es necesario pasarle el parámetro. Si bien es 0, ahora para testear necesito que el proyecto tenga ya una cantidad. Si no el test tiene que tener otras donaciones más para que tenga más de Mil pesos.

Reply:
Pero se podría hacer setAmountCollected? O no tiene ese método? ... estoy con el cel por eso pregunto... Y también por el hecho de que cuando se vuelva a poner como iba , van a romper los test

Reply:
Si está el método ese pero el tema es que ese método se le pasa al objeto tipo Project no se si está bien cuando se hace un new se le pase un setAmount... 
Porque en si un builder es para construir y no se si está bueno meterle otra cosa. 
Lo de que va a romper si no lo había pensado pero bueno se puede agregar que si es null que sea 0 el default. No sé si en java hay una manera bonita de hacerlo pero bueno lo voy a mirar

Reply:
seria como esta parte en el build() de la clase PorjectBuilder: 
if (this.factor != null)
  project.setFactor(this.factor);